mock-certify-plugin/pom.xml

```diff
@@ -53,23 +53,21 @@
         <maven.jacoco.version>0.8.11</maven.jacoco.version>
         <maven-javadoc-plugin.version>3.6.3</maven-javadoc-plugin.version>
         <kernel-keymanager-service.version>1.3.0-beta.1</kernel-keymanager-service.version>
+        <kotlin.version>2.0.0</kotlin.version>
     </properties>
-    <dependencies>
 
+    <dependencies>
         <dependency>
-            <groupId>org.projectlombok</groupId>
-            <artifactId>lombok</artifactId>
-            <version>1.18.30</version>
-            <scope>provided</scope>
+            <groupId>com.fasterxml.jackson.dataformat</groupId>
+            <artifactId>jackson-dataformat-cbor</artifactId>
+            <version>2.10.1</version>
         </dependency>
-
         <dependency>
             <groupId>io.mosip.certify</groupId>
             <artifactId>certify-core</artifactId>
             <version>0.9.0</version>
```

Comment:
update version to 0.10.0-snapshot

Reply:
Rebasing with develop has brought in 0.10.0-SNAPSHOT version for mock-certify-plugin/pom.xml